packages/twenty-server/src/engine/metadata-modules/flat-object-metadata/utils/are-flat-object-metadata-names-synced-with-labels.util.ts

```diff
@@ -0,0 +1,16 @@
+import { type FlatObjectMetadata } from 'src/engine/metadata-modules/flat-object-metadata/types/flat-object-metadata.type';
+import { computeMetadataNameFromLabel } from 'src/engine/metadata-modules/utils/validate-name-and-label-are-sync-or-throw.util';
+
+export const areFlatObjectMetadataNamesSyncedWithLabels = (
+  flatObjectdMetadata: FlatObjectMetadata,
```

Comment:
we should always Pick< FlatObjectMetadata> what we need here

Reply:
99% of the time it would result in the same call as typescript would allow a subtype out of bigger one